src/shared/local/moveDetection.ts

```diff
@@ -33,14 +38,18 @@ type StringMapsForMatches = {
 
 /** composed functions to simplified use by the shadowRepo class */
 export const filenameMatchesToMap =
-  (isWindows: boolean) =>
   (registry: RegistryAccess) =>
   (projectPath: string) =>
   (gitDir: string) =>
-  async ({ added, deleted }: AddedAndDeletedFilenames): Promise<StringMapsForMatches> =>
-    excludeNonMatchingTypes(isWindows)(registry)(
-      compareHashes(
-        await buildMaps(
+  async ({ added, deleted }: AddedAndDeletedFilenames): Promise<StringMapsForMatches> => {
```

Comment:
simplify some functions; 
we only need to do resolving once since it happens earlier